modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -184,6 +184,7 @@
     <string name="minutes_short_plural">%d mins</string>
     <string name="hours_singular">1 hour</string>
     <string name="hours_plural">%d hours</string>
+    <string name="hours_and_minutes">%d hours and %d minutes</string>
```

Comment:
Maybe we could start using plurals now that they are supported? p1714386934272659/1712851242.132719-slack-C02QANACA

Reply:
Awesome suggestion and I am glad we have this now.  I would prefer to introduce this do our app in another PR